src/automations/checkInactivity.js

```diff
@@ -137,7 +145,6 @@ async function scrapeInactiveIssues(client, references, issues) {
       owner: repoOwner, repo: repoName, number: issueNumber, per_page: 100
     });
     const com = issueComments.data.slice(-1).pop();
-
```

Comment:
Why was this line removed again?

Reply:
The presence of this line was throwing a test failure 'Trailing spaces not allowed'. This was space was probably introduced in earlier commits due to declaration of new variables.

Reply:
You're probably adding trailing whitespace to this line then (it should be only a blank line with no whitespace after). Your editor might be responsible for this issue, but this line should still not be removed nonetheless.